integration/tests/basic/test_deposit.py

```diff
@@ -190,3 +191,37 @@ def test_failed_withdraw_non_divisible_amount(
         destination_balance_after = spl_neon_token.get_balance(dest_acc.pubkey(), commitment=Commitment("confirmed"))
         with pytest.raises(AttributeError):
             _ = destination_balance_after.value
+
+    @pytest.mark.skip(reason="https://neonlabs.atlassian.net/browse/NDEV-3674")
+    def test_withdraw_wrapped(
+        self,
+        web3_client_sol: Web3Client,
+        account_with_all_tokens: LocalAccount,
+        solana_account: Keypair,
+        neon_mint: Pubkey,
+        sol_client: SolanaClient,
+    ):
+        contract, _ = web3_client_sol.deploy_and_get_contract(
+            contract="precompiled/NeonToken",
+            version="0.8.10",
+            account=account_with_all_tokens,
+        )
+
+        trx = Transaction()
+        trx.add(create_associated_token_account(solana_account.pubkey(), solana_account.pubkey(), neon_mint))
+        opts = TxOpts(skip_preflight=True, skip_confirmation=False)
+        sol_client.send_transaction(trx, solana_account, opts=opts)
```

Comment:
you can use send_tx_and_check_status_ok to be sure nothing wrong woth ata creating